target/linux/kirkwood/image/Makefile

```diff
@@ -136,6 +136,15 @@ define Device/linksys_ea4500
 endef
 TARGET_DEVICES += linksys_ea4500
 
+define Device/netgear_stora
+  DEVICE_VENDOR := Netgear
+  DEVICE_MODEL := Stora
```

Comment:
See above, I think either "Stora MS2110" or "MS2110 (Stora)" will better match the name as used by then vendor.